corehq/apps/geospatial/reports.py

```diff
@@ -0,0 +1,62 @@
+from django.conf import settings
+from django.urls import reverse
+from django.utils.translation import gettext_noop
+
+from corehq.apps.geospatial.dispatchers import CaseManagementMapDispatcher
+from corehq.apps.reports.standard import ProjectReport
+from corehq.apps.reports.standard.cases.basic import CaseListMixin
+from corehq.apps.reports.standard.cases.data_sources import CaseDisplayES
+
+
+class CaseManagementMap(ProjectReport, CaseListMixin):
+    name = gettext_noop("Case Management Map")
+    slug = "case_management_map"
+
+    base_template = "geospatial/map_visualization_base.html"
+    report_template_path = "map_visualization.html"
+
+    dispatcher = CaseManagementMapDispatcher
+
+    @property
+    def template_context(self):
+        # Whatever is specified here can be accessed through initial_page_data
+        context = super(CaseManagementMap, self).template_context
+        context.update({
+            'mapbox_access_token': settings.MAPBOX_ACCESS_TOKEN,
+        })
+        return context
+
+    def _get_geo_location(self, case):
+        geo_point = case['case_json'].get('commcare_gps_point')
+        if not geo_point:
+            return
+        try:
+            # Update if we need altitude and accuracy
+            lat, lon, _alt, _acc = geo_point.split(" ")
+            return {"lat": float(lat), "lng": float(lon)}
```

Comment:
It looks this ends up at `marker.setLngLat` which accepts dict with keys `lon` and `lat` per [the docs](https://docs.mapbox.com/mapbox-gl-js/api/geography/#lnglatlike-example). How come this still works?

Reply:
In that exaple they used the lon/lat pair, but the description of the LngLatLike object says that it will accept either lon/lat and lng/lat pairs

Reply:
Oh yes, thanks!